classes/PodsInit.php

```diff
@@ -126,13 +126,72 @@ public function __construct() {
 
 		self::$upgrade_needed = $this->needs_upgrade();
 
-		add_action( 'plugins_loaded', array( $this, 'plugins_loaded' ) );
-		add_action( 'plugins_loaded', array( $this, 'activate_install' ), 9 );
+		add_action( 'plugins_loaded', [ $this, 'plugins_loaded' ], 0 );
+		add_action( 'plugins_loaded', [ $this, 'activate_install' ], 9 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 85bdb2fc22a07a97f1ab8fefe7e1dcf5</sub>